ruledata.go

```diff
@@ -134,24 +133,32 @@ done:
 
 			}
 			if m.Header.Type == syscall.NLMSG_ERROR {
-				log.Println("NLMSG_ERROR\n")
+				error := int32(nativeEndian().Uint32(m.Data[0:4]))
+				if error == 0 {
+					log.Println("Acknowledged!!")
```

Comment:
We need to get rid of such strings.
can do this later in single commit and after some discussion.